src/ports/postgres/modules/crf/test/crf_test_small.sql_in

```diff
@@ -90,7 +90,7 @@
         (18,'PRP$'),(19,'RB'), (20,'RBR'),  (21,'RBS'), (22,'RP'), (23,'SYM'), (24,'TO'), (25,'UH'), (26,'VB'),
         (27,'VBD'), (28,'VBG'),(29,'VBN'),  (30,'VBP'), (31,'VBZ'),(32,'WDT'), (33,'WP'), (34,'WP$'),(35,'WRB'),
         (36,'$'),   (37,'#'),  (38,''''''), (39,'``'),  (40,'('),  (41,')'),   (42,','),  (43,'.'),  (44,':');
-	analyze crf_label;
+	analyze test_crf_label;
```

Comment:
Assuming that the table `crf_label` doesn't exist, why wasn't crf install check always red?